index.js

```diff
@@ -38,6 +39,11 @@ module.exports = function (input, opts) {
 			argFn = createAliasArg;
 		}
 
+		if (key === '_') {
```

Comment:
I think the idea is the arguments always get appended to the end of the `args` array. Doing the processing at line 42 would insert the `_` args at other places.